webpack/assets/javascripts/react_app/components/HostDetails/DetailsCard/PowerStatus/PowerStatusIcon.js

```diff
@@ -0,0 +1,29 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { PowerOffIcon } from '@patternfly/react-icons';
+import { Spinner } from '@patternfly/react-core';
+import { STATUS } from '../../../../constants';
+import '../styles.scss';
```

Comment:
Won't it be loaded in `webpack/assets/javascripts/react_app/components/HostDetails/DetailsCard/index.js`?